src/pybind/mgr/cephadm/module.py

```diff
@@ -2773,7 +2773,10 @@ def remove_osds(self, osd_ids: List[str],
 
         # trigger the serve loop to initiate the removal
         self._kick_serve_loop()
-        return "Scheduled OSD(s) for removal"
+        warning_zap = "" if not zap else ("\nExisting VG/LV won't be zapped (--zap wasn't passed).\n"
+                                          "Run the `ceph-volume lvm zap` command with `--destroy`"
+                                          " if you want them to be destroyed.")
```

Comment:
```suggestion
        warning_zap = "" if not zap else ("\nVG/LV for the OSDs won't be zapped (--zap wasn't passed).\n"
                                          "Run the `ceph-volume lvm zap` command with `--destroy`"
                                          " against the VG/LV if you want them to be destroyed.")
```
NIT, I personally think this is a bit clearer.